src/Sepes.Functions/Sepes.Functions.csproj

```diff
@@ -4,7 +4,7 @@
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <NoWarn>1701;1702;IDE0060</NoWarn>
+    <NoWarn>1701;1702;IDE0060;IDE0063</NoWarn>
```

Comment:
What warning is this?

Reply:
"Using statement can be simplified", but when auto-fixing this problem, it removes the brackets around the using statement, and  breaks the flow of execution.